packages/mergeOnGreen/README.md

```diff
@@ -0,0 +1,21 @@
+mergeOnGreen
```

Comment:
In JavaScript land, the convention would be to clal this `merge-on-green`, rather than the camel case `mergeOnGreen` :+1: